script/generate_views

```diff
@@ -140,13 +140,23 @@ def create_views_if_not_exist(views, exclude, sql_dir):
 
         project, dataset, viewname = view.split(".")
         target = f"{view}_v{version}"
-        view_query = VIEW_QUERY_TEMPLATE.format(target=target).strip()
         view_dataset = dataset.rsplit("_", 1)[0]
         full_view_id = ".".join([project, view_dataset, viewname])
         target_file = os.path.join(sql_dir, view_dataset, viewname, "view.sql")
-        full_sql = f"CREATE OR REPLACE VIEW\n  `{full_view_id}`\nAS {view_query}\n"
 
         if not os.path.exists(target_file):
+            replacements = [
+                "`moz-fx-data-shared-prod.udf.normalize_metadata`(metadata) AS metadata",
+            ]
+            table = client.get_table(target)
```

Comment:
We make this API call inside the loop so that the script can avoid making extra calls when target files already exist on disk.

Reply:
Maybe add a comment about that?